packages/cli/src/commands/deploy.ts

```diff
@@ -68,19 +68,6 @@ export default class Deploy extends Command {
 
     try {
       const projectPayload = project.synthesize()
-
-      // TODO: refactor Check construct to handle internal attributes properly
-      projectPayload.checks = Object.keys(projectPayload.checks).reduce((acc, checkLogicalId) => {
-        const check = projectPayload.checks[checkLogicalId]
-        delete check.__checkFilePath
```

Comment:
We no longer add `__checkFilePath` and `sourceFile` in `synthesize()`, so this isn't necessary.